perl-lib/OESS/lib/OESS/MPLS/FWDCTL.pm

```diff
@@ -352,7 +352,7 @@ sub _write_cache{
 sub _register_rpc_methods{
     my $self = shift;
     my $d = shift;
-
+    $self->{'logger'}->error("REGISTERIGN RPC METHODS MPLS/FWDCTL");
```

Comment:
This should be set to debug if logged at all.

Reply:
Removed